content/en/blog/2025/ambient-performance/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Istio: the most performant way to achieve network security"
+description: Istio's' ambient mode trailblazes as the highest performance security solution in the ecosystem.
+publishdate: 2025-03-12
+attribution: "John Howard (Solo.io)"
+keywords: [istio,performance,ambient]
+---
+
+Encryption in transit is a baseline requirement for almost all Kubernetes environments today, and forms the foundation of a zero-trust security posture. However, when it comes to security, everyone *wants* more of it, in theory.
+The problem, however, is that security doesn’t come for free – it is always a tradeoff between complexity, user experience, performance, and more.
+
+While most Cloud Native users will know of Istio as a service mesh, providing advanced HTTP functionality, it can also serve the role of providing a foundational network security layer. When we set out to build [Istio's ambient mode](/docs/overview/dataplane-modes/#ambient-mode), these two layers are explicitly split. One of our primary objectives was to be able to offer security (and a long list of [other features](/docs/concepts/)!) *without* compromise.
+
+With ambient mode, Istio is now the highest bandwidth way to achieve a secure zero-trust network in Kubernetes.
+First lets look at some results before we dive into the how and why.
```

Comment:
```suggestion
While most Cloud Native users will know of Istio as a service mesh, providing advanced HTTP functionality, it can also serve the role of providing a foundational network security layer. When we set out to build [Istio's ambient mode](/docs/overview/dataplane-modes/#ambient-mode), these two layers were explicitly split. One of our primary objectives was to be able to offer security (and a long list of [other features](/docs/concepts/)!) without compromise.

With ambient mode, **Istio is now the highest-bandwidth way to achieve a secure zero-trust network in Kubernetes**.

Lets look at some results before we dive into the how and why.
```